scripts/clean.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
What's the purpose of this file?

Reply:
clean everything in one command. I can git ignore the directory if you don't see it useful

Reply:
Mostly curious what workflow inspired you to add it

Reply:
Found myself needing to uninstall/reinstall deps, removing node_modules and `dist`s, etc. so this is a handy script I grabbed. The reason I'm doing uninstalls/reinstalled and all that stuff is cause I'm jumping between prs and deps are changing vastly across those prs. Should be temporary